src/openjd/sessions/_action_filter.py

```diff
@@ -36,8 +37,10 @@ class ActionMessageKind(Enum):
 openjd_env_actions_filter_matcher = re.compile(openjd_env_actions_filter_regex)
 
 # A regex for matching the assignment of a value to an environment variable
-envvar_set_regex = "^[A-Za-z_][A-Za-z0-9_]*" "=" ".*$"  # Variable name
-envvar_set_matcher = re.compile(envvar_set_regex)
+envvar_set_regex_str = "^[A-Za-z_][A-Za-z0-9_]*" "=" ".*$"  # Variable name
+envvar_set_regex_json = '^(")?[A-Za-z_][A-Za-z0-9_]*' "=" ".*$"  # Variable name
```

Comment:
Based on how it's used, I suspect that it won't actually matter, but I think that the regex here is intended to be more along the lines of:
```suggestion
envvar_set_regex_json = '^"[A-Za-z_][A-Za-z0-9_]*' "=" '.*"$'  # Variable name
```

i.e.
1. No `?` for the leading double-quote; and
2. Including the terminating double-quote.

Reply:
The ``(")?`` is for checking open and (mandatory) close quotes that's required for regex validation for a json. I tried with your suggestion as well but that doesn't work the same way for incorrectly formatted jsons

Reply:
Ah, right, incorrectly formatted JSON. Sounds good. We'll keep it then

Reply:
Don't we need the terminating double-quote? It looks like we're currently relying on `json.loads` to throw an error, but maybe that's fine
```suggestion
envvar_set_regex_json = '^(")?[A-Za-z_][A-Za-z0-9_]*' "=" '.*"$'  # Variable name
```

Reply:
Yeah relying on a combination - regex first for determing whether it's a json or str value and then using ``json.loads`` to verify the json itself should make it full proof